cheatsheet.cheatmd

```diff
@@ -4,9 +4,10 @@
 
 ```elixir
 def live_select(%{field: %Phoenix.HTML.FormField{} = field} = assigns) do
-  assigns = assign(assigns, :errors, Enum.map(field.errors, &translate_error(&1)))
-  
-  live_select_opts = assigns_to_attributes(assigns, [:errors, :label])
+  assigns =
+    assigns
+    |> assign(:errors, Enum.map(field.errors, &translate_error(&1)))
+    |> assign(:live_select_opts, assigns_to_attributes(assigns, [:errors, :label, :size]))
```

Comment:
`:size` can be removed, right?

Reply:
Yep, that's was my internal attr. Removed now. Nice one spotting it!